src/assets/images/hud/spritesheet.json

```diff
@@ -0,0 +1,289 @@
+{
```

Comment:
I only need "frame" and "size" data from this. I can tweak the generator script to only give me this if need be.